src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/LoadCoinviewRule.cs

```diff
@@ -10,12 +10,37 @@
 
 namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
 {
+    [FullValidationRule]
+    public class SaveCoinviewRule : UtxoStoreConsensusRule
```

Comment:
This will also need to be added to SC. Create a rule called `SmartContractSaveCoinviewRule`